_episodes_rmd/03-r-data-analysis.Rmd

```diff
@@ -331,22 +387,22 @@ gapminder_data %>%
 > The `select()` function has a bunch of helper functions that are handy if you are working with a dataset that has a lot of columns. You can see these helper functions on the `?select` help page. For example, let's say we wanted to select the year column and all the columns that start with the letter "c". You can do that with:
 > 
 > ```{r GapSelectFancy}
-> gapminder_data %>%
+> smoking %>%
 >   select(year, starts_with("c"))
 > ```
 > This returns just the three columns we are interested in. 
```

Comment:
```suggestion
> This returns just the three columns we are interested in. Note that we had to use quotation marks for "c" because the `starts_with()` function requires a character vector in the match argument.
```